Source/Core/DiscIO/VolumeWad.h

```diff
@@ -10,6 +10,7 @@
 #include <vector>
 
 #include "Common/CommonTypes.h"
+#include "DiscIO/Blob.h"
```

Comment:
If this was included for IBlobReader, this can be replaced with a forward declaration.